src/test/kotlin/no/nav/tilleggsstonader/sak/vedtak/barnetilsyn/TilsynBarnVedtakControllerTest.kt

```diff
@@ -120,9 +124,38 @@ class TilsynBarnVedtakControllerTest(
             begrunnelse = "endre utgifter opphør",
         )
 
-        opphørVedtak(behandling, vedtak)
+        val behandlingLagreOpphør = behandling(
+            steg = StegType.BEREGNE_YTELSE,
+            status = BehandlingStatus.UTREDES,
+            revurderFra = LocalDate.of(2023, 2, 1),
+            type = BehandlingType.REVURDERING,
+        )
```

Comment:
Ta gjerne en ekstra titt på dette. Gir det mening å lage en behandling som er en revurdering uten at man har en førstegangsbehandling? Vil det påvirke flyten på noen måte?

Reply:
Egentlige ikke
Tror vi kan endre til å gjenbruke behandling som settes opp i begynnelsen på testen, men å trekke ut fagsaken sånn at vi kan gjenbruke den

Vi burde nog validere at man kun opphør hvis det finnes en `forrigeBehandleId` på en behandling

```kotlin
val fagsak = fagsak()
val behandling = behandling(fagsak = fagsak, steg = StegType.BEREGNE_YTELSE, status = BehandlingStatus.UTREDES)

....

// Den første behandlingen må markeres som ferdigstilt for å kunne opprette en revurdering
steg.utførOgReturnerNesteSteg(saksbehandling, innvilgelseDto())
            testoppsettService.oppdater(behandling.copy(status = BehandlingStatus.FERDIGSTILT))
            val revurdering = opprettRevurdering(revurderFra = LocalDate.of(2023, 1, 1))
```

Med
```kotlin
private fun opprettRevurdering(revurderFra: LocalDate) = testoppsettService.lagre(
        behandling(
            fagsak = fagsak(id = behandling.fagsakId),
            type = BehandlingType.REVURDERING,
            revurderFra = revurderFra,
            forrigeBehandlingId = behandling.id,
        ),
        opprettGrunnlagsdata = false,
    ).let { testoppsettService.hentSaksbehandling(it.id) }
    
```

Reply:
Takk for innspill. Vi har fikset her: https://github.com/navikt/tilleggsstonader-sak/pull/483/commits/3e75ec5d5f48b7d83e52200f68cc63729bcbe969